tests/dummy/app/router.js

```diff
@@ -10,7 +10,7 @@ export default class Router extends EmberRouter {
 Router.map(function () {
   nrgRoutes(this);
   this.route('validation-test');
-  this.route('master-detail', function () {
+  this.route('side-by-side', function () {
     this.route('detail', {
```

Comment:
Do you want to rename this route?

Reply:
Are you talking about the detail route?
For the dummy app specifically I'm keeping the list-detail naming convention.
We use the `side-by-side` component to implement the "list-detail" use-case.]